src/craftground/wrappers/waveform.py

```
@@ -0,0 +1,434 @@
+                low=0.0, high=np.inf, shape=(self.n_mels, time_frames), dtype=np.float32
+            )
+
+        elif self.output_format == "mfcc":
+            # MFCC: (n_mfcc, time_frames)
+            time_frames = self.buffer_samples // self.hop_length + 1
+            n_mfcc = 13  # Standard number of MFCCs
+            audio_space = gym.spaces.Box(
+                low=-np.inf, high=np.inf, shape=(n_mfcc, time_frames), dtype=np.float32
+            )
+        else:
+            raise ValueError(f"Unknown output_format: {self.output_format}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)